README.md

```diff
@@ -10,6 +10,60 @@ If there is anything you are looking for that is not covered here you should ref
 
 ----
 
+### File structure ###
+
+You should not fine multiple types (ie class, struct, enum...) in the same file, each type should have its own file.
```

Comment:
`define` not `fine`. also a `;` after `same file:

`You should not efine multiple types in the same file; each type should be declared in its own file.`